src/v3/src/util/settingsUtils.test.ts

```diff
@@ -139,4 +144,82 @@ describe('Settings Utils Tests', () => {
     };
     expect(getBackToSignInUri(widgetProps)).toBe('https://okta.okta.com/');
   });
+
+  it.each`
+    setPageTitle | brandNameProvided
+    ${undefined} | ${false}
+    ${undefined} | ${true}
+    ${true}      | ${false}
+    ${true}      | ${true}
+    ${false}     | ${false}
+    ${false}     | ${true}
+  `('should return default page title when setPageTitle is: $setPageTitle and brandName provided: $brandNameProvided', ({ setPageTitle, brandNameProvided }) => {
+    const formTitle = 'Sign In';
+    const brandName: string | undefined = brandNameProvided && 'Acme Inc.';
+    widgetProps = { brandName, features: { setPageTitle } };
+
+    let expectedPageTitle: string | undefined = brandNameProvided ? `${brandName} | ${formTitle}` : formTitle;
+    if (setPageTitle === false) {
```

Comment:
I think you can remove this logic and remove the last two assertions where `setPageTitle` = false, since the next test below tests for this case.